src/components/templates/ReactionPage/ReactionList.tsx

```diff
@@ -98,11 +90,7 @@ export const ReactionList: React.FC<ReactionListProps> = ({
         {allReactions}
       </div>
 
-      <UserProfileModal
-        userProfile={selectedUserProfile}
-        show={selectedUserProfile !== undefined}
-        onHide={() => setSelectedUserProfile(undefined)}
-      />
+      <UserProfileModal />
```

Comment:
Do you still need it here?

Isn't it covered by the one declared on the `TemplateWrapper` level?